test/chemistry/test_uccsd_hartree_fock.py

```diff
@@ -114,58 +135,16 @@ def test_uccsd_hf_aer_qasm_snapshot(self):
             self.skipTest("Aer doesn't appear to be installed. Error: '{}'".format(str(ex)))
             return
         backend = Aer.get_backend('qasm_simulator')
-        algo = VQE(self.qubit_op, self.var_form, self.optimizer, expectation=AerPauliExpectation())
-        result = algo.run(QuantumInstance(backend))
-        result = self.core.process_algorithm_result(result)
-        self.assertAlmostEqual(result.energy, self.reference_energy, places=6)
+        optimizer = SPSA(maxiter=200, last_avg=5)
+        solver = VQE(var_form=self.var_form, optimizer=optimizer,
+                     expectation=AerPauliExpectation(),
+                     quantum_instance=QuantumInstance(backend=backend))
+
+        gsc = MinimumEigensolverGroundStateCalculation(self.fermionic_transformation, solver)
+
+        result = gsc.compute_groundstate(self.driver)
+        self.assertAlmostEqual(result.energy, self.reference_energy, places=3)
+
 
-    EXCITATION_RESULTS = \
```

Comment:
Why is this deleted - was it moved somewhere else that I missed? This was assuring that UCCSD computed the right expectations

Reply:
I must have lost this test in the making on the new ones. I added it back. Should be resolved.